app/routes/events/components/EventAdministrate/Abacard.js

```diff
@@ -25,7 +37,7 @@ class Abacard extends React.Component<*, State> {
     setTimeout(() => this.setState({ showCompleted: false }), 2000);
   };
 
-  handleSelect = (result: SearchResult) => {
+  handleSelect = (result: any) => {
```

Comment:
Why could this not be of type `SearchResult`, if results in an array of `SearchResults`?